### DISCUSSION TITLE
Tenacity repo is gone. What does that mean for this project?

### DISCUSSION THREAD

**Discussion by danielhoherd** [hynek/stamina#36] [Q&A] [answered]
The tenacity repo <https://github.com/jd/tenacity> that is linked in [the tenacity docs](https://tenacity.readthedocs.io) and on the [tenacity pypi page](https://pypi.org/project/tenacity/) is gone. What does that mean for the future of this project?

**Comment by danielhoherd** [comment]
Looks like jd's GH account was banned by Github. https://nitter.net/juldanjou/status/1716468432613396695#m

**Comment by hynek** [comment, marked as answer]
Huh poor Julien hope it’s gonna get sorted out fast. As long as it’s on PyPI there’s no problem for us.

**Comment by hynek** [comment]
It’s back! 🎉
